### PR TITLE
Logout Feature

### PR DESCRIPTION
**Pull request by monocliffe**
[TPOZ-1466](https://wpengine.atlassian.net/browse/TPOZ-1466)

Added Logout functionality, including a logout button, session destroyed once its clicked and a redirect to the homepage. Homepage displays 'Logout Successful!' message once logged out.

### CODE REVIEW

File: app/views/home/index.html.erb [hunk 1]
```diff
@@ -1 +1,2 @@
-<h1>Hello World!</h1>
\ No newline at end of file
+<h1>Hello World!</h1>
+<p id="notice"><%= notice %></p>
```
Comment (JamesDominy) [inline app/views/home/index.html.erb]: Can you only get notices? What about errors, etc

Comment (monocliffe) [reply app/views/home/index.html.erb]: As far as I know, it's only the name of the variable? Maybe there's another way to do it or I'm going against convention.

Comment (JamesDominy) [reply app/views/home/index.html.erb]: Oh, I might have misread this? Is this not to display flash notices? If not, is there **always** a notice present? If there isn't, then the paragraph element `<p id="notice">` should be only be conditionally displayed

Comment (monocliffe) [reply app/views/home/index.html.erb]: No, you were correct, I didn't know there was a specific method to display things conditionally. But AFAIK it does display only if I use a flash.

I did some investigation and it seems to be conditionally displayed in any case.

**Comment by monocliffe** [comment]
~~So something weird happened above, not sure what is was but it didn't break anything?~~

~~Correction, it did break stuff but I'm not sure how to fix it. Some help would be appreciated.~~

Fixed it.

File: app/views/home/index.html.erb (hunk 1 above)
Comment (monocliffe) [reply app/views/home/index.html.erb]: I changed it to a success flash banner.

File: app/controllers/application_controller.rb [hunk 2]
```diff
@@ -9,6 +10,12 @@ def current_user
     @current_user = User.where(id: session[:user_id]).first
   end
 
+  def set_cache_buster
```
Comment (JamesDominy) [inline app/controllers/application_controller.rb:13]: I've seen you use this code before, it's probably a good idea to move it into a helper/concern/mixin module  for reusability

**Review by JamesDominy — APPROVED** [review]

File: app/controllers/application_controller.rb (hunk 2 above)
Comment (JamesDominy) [reply app/controllers/application_controller.rb:13]: Oh wait, does this PR need to be rebased?

Comment (monocliffe) [reply app/controllers/application_controller.rb:13]: As far as I know, I only use it once? But I can check and change it if need be.

Also, if its merged in does it need to be rebased?

**Pull request merged**